docker/aeternity_node_mean16.yaml

```diff
@@ -19,16 +21,17 @@ keys:
   dir: ./keys
 
 chain:
-  persist: true
+  persist: false
   hard_forks:
     "1": 0
     "2": 2
     "3": 4
-    "4": 5
+    "4": 6
 
 mining:
   autostart: true
   beneficiary: "ak_2iBPH7HUz3cSDVEUWiHg76MZJ6tZooVNBmmxcgVK6VV8KAE688"
+  beneficiary_reward_delay: 2
```

Comment:
do we need that here? just want to know what's the reason for adding this :D

Reply:
I thought it will speed up the tests, but I actually don't have the math to verify the claim.

Reply:
at least it doesn't hurt :D